backend/src/main/java/com/mdbspringboot/repository/ItemRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.mdbspringboot.repository;
+
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.data.mongodb.repository.Query;
+
+import com.mdbspringboot.model.Transcript;
+
+public interface ItemRepository extends MongoRepository<Transcript, String> {
```

Comment:
I think there should be more documentation on ItemRepository because I'm confused as to what it does

Reply:
I'll add documentation in the next push.